.claude/skills/verify/tests/test_check_branch_protection.py

```
@@ -125,6 +125,148 @@ def run_tests():
 
+    # ─── Section 6: Docs-only exemption on main ───
+    # Use a fresh repo for docs-only tests to avoid interference with staged files
+    with TempDir() as docs_dir:
+        setup_git_repo(docs_dir, branch="main")
+
+        t.section("Docs-only exemption on main (should allow)")
+
+        # New .md file — should be allowed
+        write_file(docs_dir, "README.md", "# Hello")
+        subprocess.run(
+            ["git", "add", "README.md"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)